static/js/reducers/coupons.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import {
+  REQUEST_ATTACH_COUPON,
+  RECEIVE_ATTACH_COUPON_SUCCESS,
+  RECEIVE_ATTACH_COUPON_FAILURE,
+  REQUEST_FETCH_COUPONS,
+  RECEIVE_FETCH_COUPONS_SUCCESS,
+  RECEIVE_FETCH_COUPONS_FAILURE,
+} from '../actions/coupons';
+import {
+  FETCH_FAILURE,
+  FETCH_PROCESSING,
+  FETCH_SUCCESS,
+} from '../actions';
+import type { Action } from '../flow/reduxTypes';
+
+export type CouponsState = {
+  fetchAttachStatus?:  string,
```

Comment:
What do you think about naming this `fetchPostStatus` instead of `fetchAttachStatus`? The other value is named based on the http verb, so it might be more consistent.